src/AdjustSync.cpp

```diff
@@ -156,7 +156,7 @@ void AdjustSync::HandleAutosync( float fNoteOffBySeconds, float fStepTime )
 	case AutosyncType_Tempo:
 	{
 		// We collect all of the data and process it at the end
-		s_vAutosyncTempoData.push_back( std::make_pair(fStepTime, fNoteOffBySeconds) );
+		s_vAutosyncTempoData.push_back( std::pair<float, float>(fStepTime, fNoteOffBySeconds) );
```

Comment:
We can just use emplace_back here instead of push_back.

`s_vAutosyncTempoData.emplace_back(fStepTime, fNoteOffBySeconds);`